src/Be.Vlaanderen.Basisregisters.GrAr.Common/NetTopology/GeometryCoordinateValue.cs

```diff
@@ -0,0 +1,56 @@
+namespace Be.Vlaanderen.Basisregisters.GrAr.Common.NetTopology
+{
+    using System;
+    using System.Globalization;
+
+    public abstract class GeometryCoordinateValue
```

Comment:
Instead of adding a project reference, this file was copied from src/Be.Vlaanderen.Basisregisters.GrAr.Legacy/SpatialTools/GeometryCoordinates/GeometryCoordinateValue.cs